spec/features/step_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+describe 'test with right word' do
+
+  it 'right word' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.